libwebauthn/src/fido.rs

```diff
@@ -312,6 +323,13 @@ mod tests {
             &credential_id,
             &webauthn_auth_data[55..55 + &credential_id.len()]
         );
-        assert_eq!(cose_bytes, &webauthn_auth_data[55 + credential_id.len()..]);
+        let extensions_bytes = cbor::to_vec(&extensions).unwrap();
+        assert_eq!(cose_bytes, &webauthn_auth_data[55 + credential_id.len()..webauthn_auth_data.len() - extensions_bytes.len()]);
+
+        let authdata_wrapped = cbor::to_vec(&ByteBuf::from(webauthn_auth_data)).unwrap();
```

Comment:
Can you add a comment here, basically stating shortly what we are trying to do here? Something like `// Deserialize the serialized data again and verify it's the same as the original`, but maybe better worded :sweat_smile: